schemas/subsets/main.yml

```diff
@@ -657,8 +657,8 @@ fields:
         fields:
           domain: {}
           email: {}
-          entity:
-            fields: "*"
+#          entity:
```

Comment:
should we remove this all together, or add a comment with our intention if you think there's value in keeping it commented out ?

Reply:
I've removed it now